### ISSUE TITLE
[Question] Registry Disk Cleanup not working "Error: (HTTP code 409) ... image is referenced in multiple repositories"

### ISSUE THREAD

**Issue by pixply** [caprover/caprover#1646]
**What is the problem?**
I'm trying to do a cleanup of the image files within the docker registry. It's already 150G in size and that makes backups really problematic. When I do it on the backend it results in "Unused images are deleted" but it doesn't affect the size. I checked with `docker service logs captain-captain --since 10m` and I see a lot of: 

```
Error: (HTTP code 409) conflict - conflict: unable to delete aca9bed2883e (must be forced) - image is referenced in multiple repositories
captain-captain.1.5lm7hokaxdoo@cluster1   |     at /usr/src/app/node_modules/docker-modem/lib/modem.js:315:17
captain-captain.1.5lm7hokaxdoo@cluster1    |     at getCause (/usr/src/app/node_modules/docker-modem/lib/modem.js:345:7)
captain-captain.1.5lm7hokaxdoo@cluster1    |     at Modem.buildPayload (/usr/src/app/node_modules/docker-modem/lib/modem.js:314:5)
captain-captain.1.5lm7hokaxdoo@cluster1    |     at IncomingMessage.<anonymous> (/usr/src/app/node_modules/docker-modem/lib/modem.js:286:14)
captain-captain.1.5lm7hokaxdoo@cluster1    |     at IncomingMessage.emit (events.js:412:35)
captain-captain.1.5lm7hokaxdoo@cluster1    |     at endReadableNT (internal/streams/readable.js:1334:12)
captain-captain.1.5lm7hokaxdoo@cluster1    |     at processTicksAndRejections (internal/process/task_queues.js:82:21)
captain-captain.1.5lm7hokaxdoo@cluster1    | January 24th 2023, 11:53:33.801 am    Error: (HTTP code 409) conflict - conflict: unable to delete 66bfcfd3a4b9 (must be forced) - image is referenced in multiple repositories
```

I have 14 instances of the same app for different countries running, that's true and necessary. But how to get rid of the old images then? They are all automatically deployed via webhook, so they are all on the latest release.

Caprover Version 1.10.1

**Comment by githubsaturn** [comment]
See: 
* https://github.com/caprover/caprover/issues/1574
* https://github.com/caprover/caprover/issues/1397

TLDR:
Disk clean-up is a "Docker" disk cleanup - a better version of `docker image prune --all` command in Docker.
The Registry Images are not controlled by Docker or CapRover. Docker Registry is an independent app whose job is to keep all images. If you want to remove the images, you have to either manually nuke the directory where it stores all the images, or use a Docker Registry GUI app (as mentioned in one the links above)
